src/pages/workspaces/workspace/Data.js

```diff
@@ -985,83 +985,84 @@ const WorkspaceData = _.flow(
-              href: `https://seqr.broadinstitute.org/workspace/${namespace}/${name}`,
-              style: { padding: '0.5rem' }
-            }, [icon('pop-out'), ' Analyze in Seqr']),
-            noticeForPrefix: prefix => prefix.startsWith(`${dataTableVersionsPathRoot}/`) ?
-              'Files in this folder are managed via data table versioning.' :
-              null,
-            shouldDisableEditForPrefix: prefix => prefix.startsWith(`${dataTableVersionsPathRoot}/`)
-          })],
-          [workspaceDataTypes.snapshot, () => h(SnapshotContent, {
-            key: refreshKey,
-            workspace,
-            snapshotDetails,
-            snapshotName: selectedData.snapshotName,
-            tableName: selectedData.tableName,
-            loadMetadata: () => loadSnapshotEntities(selectedData.snapshotName),
-            onUpdate: async newSnapshotName => {
-              await loadSnapshotMetadata()
-              setSelectedData({ type: workspaceDataTypes.snapshot, snapshotName: newSnapshotName })
-              forceRefresh()
-            },
-            onDelete: async () => {
-              await loadSnapshotMetadata()
-              setSelectedData(undefined)
-              forceRefresh()
-            }
-          })],
-          [workspaceDataTypes.entities, () => h(EntitiesContent, {
-            key: refreshKey,
-            workspace,
-            entityMetadata,
-            setEntityMetadata,
-            entityKey: selectedData.entityType,
-            activeCrossTableTextFilter,
-            loadMetadata,
-            deleteColumnUpdateMetadata,
-            forceRefresh
-          })],
-          [workspaceDataTypes.entitiesVersion, () => h(DataTableVersion, {
-            workspace,
-            version: selectedData.version,
-            onDelete: reportErrorAndRethrow('Error deleting version', async () => {
-              await deleteDataTableVersion(selectedData.version)
-              setSelectedData(undefined)
-            }),
-            onImport: reportErrorAndRethrow('Error importing version', async () => {
-              const { tableName } = await importDataTableVersion(selectedData.version)
-              await loadMetadata()
-              setSelectedData({ type: workspaceDataTypes.entities, entityType: tableName })
-            })
-          })],
-          [workspaceDataTypes.wds, () => wdsDataTableProvider && proxyUrlLoaded && !_.isEmpty(wdsSchema) && h(WDSContent, {
-            key: refreshKey,
-            workspaceUUID: workspaceId,
-            workspace,
-            dataProvider: wdsDataTableProvider,
-            recordType: selectedData.entityType,
-            wdsSchema
-          })]
+        Utils.switchCase(selectedData?.type, [undefined, () => Utils.cond([wdsSchemaError && isAzureWorkspace, () => div({ style: { textAlign: 'center' } }, [icon('loadingSpinner'), ' The database that powers your data tables is being created. This may take a few minutes.'])],
```

Comment:
How come we check for `wdsSchemaError`? Maybe this is a dumb question, do we expect wdsSchemaError to be true before wds is loaded?

Reply:
Hmm, my interpretation was that `wdsSchemaError` indicates that there's some problem with wds - hopefully just that the app is still deploying but potentially something else gone wrong.  It's initialized as `undefined` though which maybe is an issue?